MVC/view/CLIrefactor.py

```diff
@@ -68,7 +68,12 @@ def startGame(self):
             if '!' not in userInput:
             # controller user guess function
                 if self.controller.controllerGetPuzzleState() != 1:
-                    print("You have not started a puzzle. Type !newpuzzle to start playing, !exit to exit, !help for more options.")
+                    print('''You have not started a puzzle yet. To get started, you can type:
```

Comment:
remove the first sentence (since it's obvious) and format this properly. you can look at examples prior but so long as you use the ''', the formatting inside there is a 1 to 1 to how it would show up in the terminal